lib/ex_webrtc/dtls_transport.ex

```diff
@@ -60,19 +67,20 @@ defmodule ExWebRTC.DTLSTransport do
   end
 
   @impl true
-  def init([ice_config, ice_module, owner]) do
+  def init([ice_transport, ice_config, owner]) do
     # temporary hack to generate certs
     dtls = ExDTLS.init(client_mode: true, dtls_srtp: true)
     cert = ExDTLS.get_cert(dtls)
     pkey = ExDTLS.get_pkey(dtls)
     fingerprint = ExDTLS.get_cert_fingerprint(dtls)
 
-    {:ok, ice_agent} = ice_module.start_link(:controlled, ice_config)
+    {:ok, ice_transport_state} = ice_transport.start_link(:controlled, ice_config)
     srtp = ExLibSRTP.new()
 
     state = %{
       owner: owner,
-      ice_agent: ice_agent,
+      ice_transport: ice_transport,
+      ice_transport_state: ice_transport_state,
```

Comment:
`ice_transport_state` is very misleading, as it has nothing to do with any state.
IMO the `ice_transport` should be called maybe like `ice_transport_impl` or `ice_transport_module` and the `ice_transport_state` should be `ice_transport`.

Or maybe `ice_transport` and `ice_pid`, like in the `PeerConnection`.